src/licence-status-banner/index.jsx

```diff
@@ -69,13 +67,7 @@ class LicenceStatusBanner extends Component {
             this.renderDates()
           }
           {
-            isInactiveVersion && <Fragment>
-              <p><Snippet>invalidLicence.summary.ppl_active</Snippet></p>
-              <p><Link page="project.version.read" versionId={this.props.licence.granted.id} label={<Snippet>{'invalidLicence.view'}</Snippet>} /></p>
```

Comment:
Do we not need to link to the active version any more?

Reply:
https://github.com/UKHomeOffice/asl-pages/pull/691/files#diff-01987d26fec6f3e56dec2aa0ed6d17d2R22-R28